exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/functions/MathFunctionTests.kt

```diff
@@ -114,7 +114,7 @@ class MathFunctionTests : FunctionsTestBase() {
                         assertExpressionEqual(null, SqrtFunction(intLiteral(-100)))
                     }
                 }
-                TestDB.SQLITE, TestDB.POSTGRESQL, TestDB.POSTGRESQLNG, TestDB.ORACLE -> {
+                TestDB.POSTGRESQL, TestDB.POSTGRESQLNG, TestDB.ORACLE -> {
                     // SQLite, PSQL, Oracle fail to execute sqrt with negative value
```

Comment:
SQLite could be removed from the comment here too since it's no longer part of the branch.

Reply:
Thanks, will fix